Minor markup changes, CSS header and body styling, web font with preload

Gideon28 opened the pull request:
- Most markup changes are to add classes for styling structure.
- The header reflows to the original layout on smaller screens, but horizontal in larger viewports.
- Back to top button added.
- Google font added to the repo so that it can be preloaded and for "font-display" to be set.
- Body background is SVG so very lightweight.
- No real performance impact observed locally.

lukeb-uk commented on site/styles.css:
```diff
@@ -193,17 +272,70 @@ input[type="submit"]:hover {
+    }
+    .page-header,
+    .card {
+        box-shadow: 0 14px 20px rgba(0,0,0,.3);
```
This doesn't really seem to impact anyyhing due to the dark background, do we still need this?

Gideon28 replied:
No, you're right, it's left over from before I changed the background... will remove.

lukeb-uk commented on site/styles.css:
```diff
@@ -193,17 +272,70 @@ input[type="submit"]:hover {
+    opacity: 1;
+    pointer-events: all;
+    position: fixed;
+    bottom: 6rem;
```
This feels a little high up the page when I'm using it, could we knock it down a little?

lukeb-uk commented on site/styles.css:
```diff
@@ -193,17 +272,70 @@ input[type="submit"]:hover {
     margin-right: 0;
 }
 
+.back-to-top {
```
Could we add a border or something around the button? On mobile we can currently get issues like this: 

![deploy-preview-3--leedsjs netlify com_events_2019-05-29_(Galaxy S5)](https://user-images.githubusercontent.com/4780756/57585563-d3b79c00-74e1-11e9-97e3-f1062bda204b.png)

Gideon28 replied:
I've made it a white circle with box-shadow instead... it should still contrast enough, see what you think :)

lukeb-uk commented:
Thank you so much for this! I have added a few comments inline and have a couple more:

Can we add some spacing around the nav in mobile mode? It can jam right up to the edge sometimes:
![deploy-preview-3--leedsjs netlify com_events_2019-05-29_](https://user-images.githubusercontent.com/4780756/57585614-59d3e280-74e2-11e9-9cce-8a2fdfdb5d10.png)

Could you add the font to the serviceworker and bump the version?
https://github.com/LeedsJS/website/blob/master/site/sw.js#L45

Your work is hugely appreciated, you've made the site look way better!

Gideon28 commented:
My pleasure, I've addressed the comments you made and have updated the service worker - I've only added the woff2 file, I wasn't sure if they all need adding... feel free to amend as needed.

lukeb-uk commented:
Awesome!

You might have noticed that I've added you as a contributer on this repo, so you won't have to work with your fork anymore!
